recipe/meta.yaml

```diff
@@ -11,17 +12,17 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  script: POETRY_DYNAMIC_VERSIONING_BYPASS={{ version }} {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
@Marie59 looks like this package uses git info, which is not present in the tarball, and that is why we get 0.0.0. See https://gitlab.ifremer.fr/cerbere/cerbere/-/blob/main/pyproject.toml?ref_type=heads#L41

I like setuptools-scm with the "write-to-file" option b/c it avoids this isse. I don't use poetry. With that said, here in conda-forge we can use `POETRY_DYNAMIC_VERSIONING_BYPASS` to fix this.

Reply:
Oh okay ! I'm gonna write that down ! Thanks for the info !

Reply:
BTW, this is a bit outdated, but I try to collect some "okish" packaging practices example in https://github.com/ioos/ioos-python-package-skeleton.